_docs/reference/config/kubernetes-proxy-injection.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Kubernetes sidecar proxy injection
+overview: Kubernetes sidecar proxy injection
+
+order: 15
+
+layout: docs
+type: markdown
+---
+
+Each pod in the Kubernetes cluster must be running an Istio compatible 
+sidecar. The following sections describe two ways of injecting the Istio 
+sidecar into a pod: manually using istioctl `kube-inject` or automatically 
+using the Istio sidecar injector webhook. 
+
+Manual injection modifies the controller configuration, e.g. deployment. It 
+does this by modifying the pod template spec such that *all* pods for that
+deployment are created with the injected sidecar. Adding/Updating/Removing
+the sidecar requires modifying the entire deployment.
+
+Automatic injection injects at pod creation time. The controller resource is 
+unmodified. Sidecars can be selectively added/updated/removed by (re)starting 
+pods after modifying the injection configuration.
+
```

Comment:
I think mention restarting pods can be confusing as kubectl doesn't have a restart or start cmd.

Reply:
Maybe we could mention rollout a new ReplicaSet or delete the pods manually instead